src/components/SearchableDropdownList.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import SearchField from './SearchField';
+import SearchableDropdownListItem from './SearchableDropdownListItem';
+
+const SearchableDropdownList = ({ places, handleListItemClick }) => {
+  const [filteredSearches, setFilteredSearches] = useState([]);
+  const [searchWord, setSearchWord] = useState('');
+
+  useEffect(() => {
+    setFilteredSearches(places);
+  }, [places]);
+  
+  const handleChange = (e) => {
+    setSearchWord(e.target.value);
+
+    const currentFiltered = places.filter((place) => place.includes(e.target.value));
+    setFilteredSearches(currentFiltered);
+  };
+
+  return (
+    <div className="dropdown__list">
+      <SearchField
+        placeholder="Search..."
+        handleChange={handleChange}
+        value={searchWord} />
+      {filteredSearches.map((place) => (
```

Comment:
instead you can: 
`places.filter(place => place.includes(searchWord)).map(place => ...)`

later if you want you can use `useMemo` to memoize filtering